testutils/factories/sentReferral.ts

```diff
@@ -11,6 +11,7 @@ const exampleReferralFields = () => {
     createdAt: '2020-12-07T20:45:21.986389Z',
     completionDeadline: '2021-04-01',
     serviceProvider: {
+      id: '1234',
```

Comment:
aha - i imagine this factory class is used in the contract tests, so this addition may be all that's required to validate it. let's double check that though.